app/controllers/dashboards_controller.rb

```diff
@@ -14,7 +15,7 @@ def set_happy_things
     friend_ids = current_user.friends.pluck(:id) + current_user.inverse_friends.pluck(:id) + [current_user.id]
     @happy_things = HappyThing.where(user_id: friend_ids)
     @happy_things_today = happy_things_by_period(Date.today..Date.tomorrow, friend_ids)
-    @happy_things_of_the_last_two_days = happy_things_by_period((Date.today - 2.days)..Date.today.end_of_day, friend_ids)
+    @happy_things_of_the_last_two_days = happy_things_by_period((Date.today - 1.days)..Date.today.end_of_day, friend_ids)
```

Comment:
The instance variable name is `of_the_last_two_days` but the query is only `1`

Reply:
Yes bc, 24 hours has both yesterday and today so it looks like two days even though the query is one day 🤓